examples/mockups/index.html

```diff
@@ -146,23 +146,23 @@ <h2 class="ui header">How it works</h2>
                     <img src="images/infographic-01.png">
                   </div>
                   <p>
-                    Write your documentation easily, using .md or .rst, directly in your own IDE, as you work on what maters: your project.
+                    Write your documentation easily, using .md or .rst, directly in your own IDE, as you work on what matters: your project.
                   </p>
                 </div>
                 <div class="column center aligned">
                   <div class="ui image">
                     <img src="images/infographic-02.png">
                   </div>
                   <p>
-                    Upload your work to your favorite source code repository hosting service. We support GitHub, GitLab, BitBucket and more.
+                    Upload your work to your favorite code hosting service. Use GitHub, GitLab, BitBucket, or your own.
```

Comment:
Tried to make this a bit shorter, I think the meaning is retained.

Reply:
It works. I like it.

Reply:
As a step that describes what RTD does, does this step seem necessary to call out?

I feel like steps should be something like:

* You write docs in your tools
* We build HTML/etc docs and host them
* Your project has search, managed versions, private authentication, etc. Emphasis on "you don't have to do anything here, RTD does all of this for you".

I suppose going back a bit, we introduced RTD to the user with "We do the rest" but then haven't really sold the user on what is included in "the rest".

Reply:
Also though, I suppose this could be tweaked slightly too, to describe *what RTD does* instead of *what the user is doing*. "RTD watches your repository and rebuilds your documentation any time you merge a change" -- though it would probably help to be a bit less technical here.

Reply:
Using that sentence, we can come back to it later.

Reply:
We should create issues for anything that we're punting on. I don't want to lose this conversation, these blocks will require some attention.

Reply:
Opened https://github.com/readthedocs/site-community/issues/32 to continue this discussion, I'd consider this a fairly important change.